setup.py

```diff
@@ -22,13 +22,9 @@
     install_requires=[
         "contextdecorator>=0.10.0",
         "inflection>=0.3.1",
-        # There is a competing upper version requirement for tornado
-        # between jaeger and open-tracing, the latter of which allows up to 6
-        # while jaeger 4.0 only allows up to 5.
-        "tornado<5",
-        "jaeger-client>=4.0.0",
```

Comment:
below I am bumping min required versions of jager and opentracing to latest ones, ensuring that they both now support `tornado<6`. given that they share this requirement, we no longer need to pin tornado ourselves explicitly either.

cc/ @cdm-ium  / @GabeLambda any quick way to validate what I am saying works other than making a release and just trying this out on a live service?